index.js

```diff
@@ -53,7 +53,9 @@ function writeMeaningToFile(inputStream, filename) {
         toWrite += "    - " + meaning.partOfSpeech + "\n";
         meaning.definitions.forEach(definition => {
             toWrite += "        - " + definition.definition + "\n";
-            toWrite += "            - ex: " + definition.example + "\n";
+            if(definition.example){
```

Comment:
please do a strict check for undefined (triple equals)

Reply:
@ashawe  done